motr/ut/protection_info_checks.c

```diff
@@ -0,0 +1,340 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2016-2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Since it is a new file, copyright year should be `2021`.
`Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates`

Reply:
For other modified files, please change 2020 -> 2021

Reply:
Done.